.github/workflows/tag-publish.yml

```diff
@@ -45,8 +45,7 @@ on:
 
 env:
   SERVICE_NAME: ${{ github.event.repository.name }}
-  GOOGLE_PROJECT: terra-kernel-k8s
-  GKE_CLUSTER: terra-kernel-k8s
```

Comment:
What happens without the GKE cluster?

Reply:
It was unused. I don't know why it was there.